### DISCUSSION TITLE
The `--scope` option to `spack config` is not appropriate for several informational subcommands

### DISCUSSION THREAD

**Discussion by greenc-FNAL** [spack/spack#41501] [Ideas]
Specifically:

* `blame`
* `list` (would anyway suggest renaming to `list-sections`)
* [`list-scopes`](https://github.com/spack/spack/pull/41455)

I would suggest either:

1. Making the `--scope` option mutually-exclusive with the inapt subcommands.
2. Deprecating the command-level option and adding `--scope` as a common argument to applicable subcommands.
3. A new command (`spack config-info`, say) for the inapt subcommands.

Comments?

**Comment by tgamblin** [comment]
I think (2) makes the most sense but then again if the option isn't relevant do we really need to move it?

What's the harm in keeping it at the `spack config` level?
